todos/app/resources/users/resource.rb

```diff
@@ -6,6 +6,14 @@ class Resource < ApplicationResource
     authorize :admin, on: :destroy
     authorize :admin, :supervisor, on: %i[create update]
 
+    sort_by :custom_sort
+
     public_actions :index, :show
+
+    extend_model do
+      def custom_sort
+        order(:email)
```

Comment:
A ridiculous example just to show it's possible, even though it's not necessary.